jobs/garden/spec

```diff
@@ -84,3 +84,12 @@ properties:
   garden.btrfs_store_size_mb:
     description: "Size in MB of the BTRFS backing store for graph storage (-1 to use all available space)"
     default: -1
+
+  env.http_proxy:
+    description: Http proxy to connect to docker hub 
+
+  env.https_proxy:
+    description: Https proxy to connect to docker hub
```

Comment:
These description strings aren't entirely accurate. When we set e.g. `HTTP_PROXY`, we're setting it for the entire `garden-linux` server process, so any time that process sends an http request it is going to use what's set here. It would be best if the documentation reflected that.